### PR TITLE
feat: sync all supported langs

### PR DESCRIPTION
**Pull request by OmarIthawi**
## Description
We're syncing the languages from the old project ( https://app.transifex.com/open-edx/edx-platform/ ) to the new project ( https://app.transifex.com/open-edx/openedx-translations/ ) on Transifex.

This pull request includes all supported languages in the Open edX project: 

 - https://openedx.atlassian.net/wiki/spaces/COMM/pages/3157524644/Translation+Working+Group#The-following-is-a-table-of-the-latest-list-of-languages-supported--by-the-Translation-Working-Group

## Future work
Another update is required to include all the languages that might be supported in the future. [Related discussion on Slack](https://openedx.slack.com/archives/C037XDB9KN1/p1696870236377009).

## Reference 

This contribution is part of the [FC-0012 project](https://openedx.atlassian.net/l/cp/XGS0iCcQ) which is sparked by the [Translation Infrastructure update OEP-58](https://open-edx-proposals.readthedocs.io/en/latest/architectural-decisions/oep-0058-arch-translations-management.html#specification).

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @OmarIthawi!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


This is currently a draft pull request. When it is ready for our review and all tests are green, click "Ready for Review", or remove "WIP" from the title, as appropriate.

File: Makefile [hunk 1]
```diff
@@ -3,8 +3,9 @@ sync_translations sync_translations_github_workflow
 
 
 # Default languages for the sync_translations.py file
-# Eventually, this should be all the languages Open edX supports. We're starting with a subset for now.
-export TX_LANGUAGES := ar,de,fr_CA
+# This includes languages which are translatie
+# This list should be all the languages Open edX supports until the migration is complete.
+export TX_LANGUAGES := ar,da,de_DE,el,es-149,es_ES,fr_CA,hi,id,it_IT,pt_BR,pt_PT,ru,th,tr_TR,uk,zh_CN
```
Comment (shadinaif) [inline Makefile:8]: Hebrew is in [the table](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3157524644/Translation+Working+Group#The-following-is-a-table-of-the-latest-list-of-languages-supported--by-the-Translation-Working-Group) but without a code
```suggestion
export TX_LANGUAGES := ar,da,de_DE,el,es-149,es_ES,fr_CA,hi,he,id,it_IT,pt_BR,pt_PT,ru,th,tr_TR,uk,zh_CN
```

**Review by shadinaif — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (shadinaif) [reply Makefile:8]: I've fixed the table

**Comment by openedx-webhooks** [comment]
@OmarIthawi 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/openedx-translations&entry.867055334=https://github.com/openedx/openedx-translations/pull/1482&entry.1484655318=https://github.com/OmarIthawi&entry.752974735=2023-10-09+17:00&entry.1917517419=2023-10-10+08:27&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**